### DISCUSSION TITLE
I can help, just need to know what needs to get done.

### DISCUSSION THREAD

**Discussion by byblakeorriver** [rrevenantt/antlr4rust#79] [General]
Hey there @rrevenantt :wave:, I am able to assist on this project in my free time.
Is there any issues that you would like me to look into?
Thanks and have a great day!

**Comment by rrevenantt** [comment]
Thanks, on the top of my head I can only suggest a task of rebasing generator for this target to latest antlr version. They have recently changed test framework significantly so that's not a trivial job.

**Reply by byblakeorriver** [reply]
Awesome! I will take a look and see if it something I can handle. I will let you know how it goes!

**Comment by byblakeorriver** [comment]
Not trivial, but I think I can swing it. I see what it will take and what has to happen. Now to just find the time. I will update as I make progress.
